patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1138,7 +1138,7 @@
 +     * @param neighborBlock The updated block.
 +     * @param neighborPos The updated block's position.
 +     */
-+    public void func_190962_b(IBlockState p_190962_1_, World p_190962_2_, BlockPos p_190962_3_, Block p_190962_4_, BlockPos p_190962_5_){}
++    public void onNeighborChangeObserver(IBlockState state, World world, BlockPos pos, Block neighborBlock, BlockPos neighborPos){}
```

Comment:
I think naming this method `onNeighborChangeObserved` or `observeNeighborChange`.